src/Bridge/Doctrine/Orm/ItemDataProvider.php

```diff
@@ -34,21 +31,18 @@
 class ItemDataProvider implements ItemDataProviderInterface
 {
     private $managerRegistry;
-    private $propertyNameCollectionFactory;
-    private $propertyMetadataFactory;
+    private $identifierManager;
     private $itemExtensions;
 
     /**
-     * @param ManagerRegistry                        $managerRegistry
-     * @param PropertyNameCollectionFactoryInterface $propertyNameCollectionFactory
-     * @param PropertyMetadataFactoryInterface       $propertyMetadataFactory
-     * @param QueryItemExtensionInterface[]          $itemExtensions
+     * @param ManagerRegistry               $managerRegistry
+     * @param IdentifierManagerInterface    $identifierManager
+     * @param QueryItemExtensionInterface[] $itemExtensions
      */
-    public function __construct(ManagerRegistry $managerRegistry, PropertyNameCollectionFactoryInterface $propertyNameCollectionFactory, PropertyMetadataFactoryInterface $propertyMetadataFactory, array $itemExtensions = [])
```

Comment:
lol @Simperfit  still we don't need the factories in the data provider but they're needed to get the identifiers. What would be the best to inject those?